source/developers/how-tos/ongoing-maintainers-tasks.rst

```diff
@@ -35,15 +35,27 @@ GitHub Notifications
 
 TBD: How to set up a group, what the notification settings should look like for that group
 
-  Responding to Issues
+Responding to Issues
 ********************
 
 TBD: Explain what a maintainer's responsibilities are around issue response time
 
 Responding to PRs
 *****************
 
-TBD: Explain what a maintainer's responsibilities are around PR review/response/merge/decline time
+Our goal for PR response time is that they are "triaged" within a week of receipt. This does not mean that review happens within a week, but that the PR is acknowledged and scheduled for review in line with the maintainers other priorities.  This response time goal is onging.  So when an author responds to feedback, the goal is the same.
+
+One of the scarest resources of an open-source project is contributor and maitainer time.  It is ideal that once a review starts, that it can be finalized as a single piece of work, say within the boardaries of a single sprint.
+
+As a maintainer, your goal should be keeping the time to review and finalize a PR predictable and as short as reasonable.  Finalizing a PR can mean rejecting it. Knowing that a PR won't be accepted and why provides the contributor appropriate feedback and let's them decide what to do next.
+
+If your team schedules reviews for future sprints, providing that information to the author is helpful to set their expectations and plan accordingly. Leave a comment indicating when you expect the review to occur and ensure that the status on the contributions board is "Scheduled for Eng Review."
+
+When reviewing a PR, we want to focus on reducing the back-and-forth that increases lead time.  In our globally distributed project, a comment and response cycle can easily take 24 hours.  Ideally, both contributor and reviewer can both focus their attention on finalizing a PR at the same time.
```

Comment:
> Ideally, both contributor and reviewer can both focus their attention on finalizing a PR at the same time.

I'm not sure the purpose of this sentence. What action or knowledge am I supposed to take from it? Should reviews _only_ occur if this is the case? What if it's not? I think it's obvious that this is the ideal case but I'm missing the goal of including this statement here.

Reply:
If a reviews two weeks after the PR comes in, b has moved on, b finishes their sprint and returns to the PR.  A is now in their sprint and doesn't get back to it for another week, and so on.  Ideally, everyone can focus on completing the review in the same timeslice and avoid delays.

Reply:
OK, I'm not entirely sure I understand my original nitpick here anyway. Makes sense.